### PR TITLE
H.E.C.C. suit resprite

### PR DESCRIPTION
**Pull request by McRamon**
## What Does This PR Do
Resprites H.E.C.C suit and helmet. Coloring ability removed. It can be added later but that would require a coder help.

## Why It's Good For The Game
The sprite is unbelievably bad

## Images of changes
image
image

## Testing
Tested on all species

## Changelog
:cl:
imageadd: New sprites for HECC suit
/:cl:

### CODE REVIEW

**Comment by spacegeese** [comment]
SAY IT WITH ME NOW. 
Let's.
Fucking.
image
GOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO

**Comment by 1080pCat** [comment]
Vox hecc suit sprites lets GOOOOO

**Comment by Henri215** [comment]
image
image
You got conflicts

File: code/modules/mining/equipment/explorer_gear.dm [hunk 1]
```diff
@@ -53,12 +53,17 @@
 	slowdown = 0
 	armor = list(MELEE = 120, BULLET = 35, LASER = 25, ENERGY = 25, BOMB = 150, RAD = INFINITY, FIRE = INFINITY, ACID = INFINITY)
 	allowed = list(/obj/item/flashlight, /obj/item/tank/internals, /obj/item/resonator, /obj/item/mining_scanner, /obj/item/t_scanner/adv_mining_scanner, /obj/item/gun/energy/kinetic_accelerator, /obj/item/pickaxe, /obj/item/melee/spellblade)
+	hide_tail_by_species = list("Vox" , "Vulpkanin" , "Unathi" , "Tajaran")
```
Comment (Henri215) [inline code/modules/mining/equipment/explorer_gear.dm:56]:
```suggestion
	hide_tail_by_species = list("Vox", "Vulpkanin", "Unathi", "Tajaran")
```

**Review by Henri215 — APPROVED** [review]
Lewcs Good To Me

File: code/modules/mining/equipment/explorer_gear.dm [hunk 2]
```diff
@@ -53,12 +53,17 @@
...
 	armor = list(MELEE = 120, BULLET = 35, LASER = 25, ENERGY = 25, BOMB = 150, RAD = INFINITY, FIRE = INFINITY, ACID = INFINITY)
 	allowed = list(/obj/item/flashlight, /obj/item/tank/internals, /obj/item/resonator, /obj/item/mining_scanner, /obj/item/t_scanner/adv_mining_scanner, /obj/item/gun/energy/kinetic_accelerator, /obj/item/pickaxe, /obj/item/melee/spellblade)
+	hide_tail_by_species = list("Vox", "Vulpkanin", "Unathi", "Tajaran")
 
+	sprite_sheets = list(
+		"Vox" = 'icons/mob/clothing/species/vox/suit.dmi',
+		"Drask" = 'icons/mob/clothing/species/drask/suit.dmi',
+		"Grey" = 'icons/mob/clothing/species/grey/suit.dmi',
+		"Tajaran" = 'icons/mob/clothing/species/tajaran/suit.dmi',
+        "Vulpkanin" = 'icons/mob/clothing/species/vulpkanin/suit.dmi',
```
Comment (SteelSlayer) [inline code/modules/mining/equipment/explorer_gear.dm:63]: CI failure here: 
`
code/modules/mining/equipment/explorer_gear.dm:63: Space indentation detected, please use tab indentation.
`

**Review by S34NW — APPROVED** [review]

**Pull request merged**